chapters/annotations.tex

```diff
@@ -72,6 +72,19 @@ \subsection{Class Description and Revision History}\label{annotation-info-revisi
 <a href="modelica:/MultiBody.Joints.Revolute#info">Revolute</a>
 \end{lstlisting}
 
+\subsubsection{Style Sheet}
+
+A top-level package may contain an annotation \lstinline!styleSheet="modelica:/Package/Resources/style.css"!, which provides a cascading style sheet.
+The given style sheet is used when displaying the \lstinline!info! and \lstinline!revisions! annotations found in the \lstinline!Documentation! annotation.
```

Comment:
```suggestion
The given style sheet is used when displaying the \lstinline!info! and \lstinline!revisions! annotations found in the \lstinline!Documentation! annotations of the package.
```
The reason it said "annotations" is that there are (normally) multiple such annotations in the package, so clarifying that with "of the package"